deploy/data_exporter/data_export_function/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM public.ecr.aws/docker/library/ubuntu:24.04
+
+RUN apt update && \
+    apt install -y postgresql-client-16 python3.12 python3-pip curl unzip && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt .
+RUN pip3 install --break-system-packages -r requirements.txt
+RUN pip3 install --break-system-packages awslambdaric
+
+COPY . .
+
+ENTRYPOINT ["python3", "-m", "awslambdaric" ]
```

Comment:
For tidyness, I'd probably bung this in a venv instead of using `--break-system-packages`
Untested, but if you put everything in a venv, you should then be able to edit your entry point to `ENTRYPOINT ["venv/bin/python3", "-m", "awslambdaric" ]` and you're done :crossed_fingers: